deployment/helm/node-feature-discovery/templates/cert-manager-certs.yaml

```diff
@@ -60,8 +60,8 @@ spec:
   dnsNames:
   - {{ include "node-feature-discovery.fullname" . }}-topology-updater.{{ include "node-feature-discovery.namespace" .  }}.svc.cluster.local
   issuerRef:
-    name: nfd-ca-issuer
-    kind: Issuer
+    name: {{ default "nfd-ca-issuer" .Values.tls.certManagerCertificate.issuerName }}
+    kind: {{ default "Issuer" ( kindIs "invalid" .Values.tls.certManagerCertificate.issuerName | ternary "" .Values.tls.certManagerCertificate.issuerKind ) }}
```

Comment:
This is quite cryptic to read, at least for me 😇

How about a bit longer but more explicit:

```sugggestion
    {{- if and .Values.tls.certManagerCertificate.issuerName .Values.tls.certManagerCertificate.issuerKind }}
    kind: {{ .Values.tls.certManagerCertificate.issuerKind }}
    {{- else }}
    kind: Issuer
    {{- end }}
```

Reply:
Got it.  Updated the PR to use more simplified logic with regards to the issuer kind.